app/assets/javascripts/application.js

```diff
@@ -40,8 +40,16 @@
 //= require controllers/ops/log_collection_form_controller
 //= require controllers/ops/tenant_form_controller
 //= require controllers/ops/tenant_quota_form_controller
+//= require controllers/container_dashboard/util/util-module
+//= require controllers/container_dashboard/util/dashboard-utils
+//= require controllers/container_dashboard/util/chart-data-mixin
+//= require controllers/container_dashboard/util/chart-config-util
 //= require controllers/container_dashboard/container_dashboard_controller
-//= require controllers/container_dashboard/container_dashboard_utils
+//= require controllers/container_dashboard/card/card.module
+//= require controllers/container_dashboard/card/heatmaps/heatmaps-card-directive
+//= require controllers/container_dashboard/charts/charts-module
+//= require controllers/container_dashboard/charts/heatmap/heatmap-directive
+//= require controllers/container_dashboard/charts/heatmap/heatmap-legend-directive
```

Comment:
Yes, we will probably have to look into some wildcard support, or generating that file or something like that, it's not realy fun to keep every single file there... But for now this is indeed the correct place.

As for high/voltage .. by one of these I assume you mean a haml file that's with some directive? Not a js file, right?